Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM adoptopenjdk
```

Comment:
This should probably call for a specific version tag, for repeatability.
Example: adoptopenjdk:11-jre-hotspot